### DISCUSSION TITLE
Adding "mailto" link to a social icon

### DISCUSSION THREAD

**Discussion by tobythesharpeagle** [gethinode/hinode#311] [Help] [answered]
I've added an envelope icon in the footer along with other social media icons (i.e., instagram, LinkedIn, Github, etc.). In the `menu.toml` file, I've added `"mailto:<email address>"` as the url, as shown below. 

 `[[social]]`
 `name = "Email"`
  `pre = "<i class=\"fas fa-envelope fa-2x\"></i>"`
  `url = "mailto:blah@google.com"`
  `weight = 10`

When the site generates, a slash is added to the front of this url so it would create a `"<website address>/mailto:<email address>"` instead of just `"mailto:<email address>"`. Hence, this would lead to a 404 page not found. 

What do I need to edit and where so that the url generated for this email social icon is simply `"mailto:<email address>"`?

Thank you in advance!

**Comment by markdumay** [comment, marked as answer]
I created a [test repo](https://github.com/markdumay/hinode-social) with the `mailto` link. I could not reproduce the error, however, I did find some code that might cause your issue. The file `layouts/partial/footer/social.html` renders the social links. I copied the file to the test repo. I suggest to  remove the `relLangURL` instruction - this is not applicable to external URL's anyways.

**Current code:**
```html
<a href="{{ .URL | relLangURL }}" [...]>
    {{ .Pre | safeHTML }}
</a>
```

**New code:**
```html
<a href="{{ .URL }}" [...]>
    {{ .Pre | safeHTML }}
</a>
```

**Reply by tobythesharpeagle** [reply]
It was the post-processing of the URL. thank you!

**Reply by markdumay** [reply]
Great! I added the changes to the core library too, available in [release v0.15.1](https://github.com/gethinode/hinode/releases/tag/v0.15.1).
